### ISSUE TITLE
Nodejs auto update version

### ISSUE THREAD

**Issue by rattar001** [nodejs/help#3615]
### Details

I have install nodejs version 16.13.0 version in the morning
and after restart my computer I found current version is 17.1.0
I don't want to use version 17.1.0 but I don't know why version is always upgrade? 
and how I can fixed this

### Node.js version

16.13.0 to 17.1.0

### Operating system

windows 10

### Scope

Installation

### Module and version

Not applicable.

**Comment by mhdawson** [comment]
How did you install Node.js ? I don't think our installer will auto update you.

**Comment by dsankarraj** [comment]
my entire team facing the same issue. installed 14.14 version. but it is frequently auto upgraded to 17.0.1

**Comment by dsankarraj** [comment]
i installed nodejs by using msi installer

**Comment by dsankarraj** [comment]
i am using visual studio 2019 to develop my angular application. In 6 days the nodejs was auto upgraded 3 time

**Comment by casually-creative** [comment]
Getting the same issue since the latest windows update. Now, everytime I restart my PC, Node is updated to version 19.0.1 (current latest), while I keep manually reinstalling 18.12.1 (currently LTS). It keeps breaking things.

**Comment by zgenov** [comment]
A possible reason is Chocolatey.In the `Tools for Native Modules` there is a checkbox

**Comment by molog** [comment]
Same issue happened recently to me too. It automatically upgraded from v16 to v19.  
Windows 11.
Chocolatey is not installed. Visual studio is not installed.

**Comment by mhdawson** [comment]
@nodejs/platform-windows anybody have any insight?

**Comment by ivan-markov-666** [comment]
I met the same issue.
Windows 10 64-bit pro
NodeJS 16.14.2 - after Windows restart, the version is updated to the last one (not the LTS version, to the final version).

**Comment by KunwarVSuryavanshi** [comment]
Happened with me also. Didn't run any install or update command in any of the repo, but on restarting my system it was updated to v19 from v16, and yes Chocolatey is not installed.

**Comment by its-saeed** [comment]
Same for me. I install v16.19.0 but when I suspend my laptop for a while and start my work again it's updated to v18.7.0

**Comment by dsankarraj** [comment]
Hi After updated the angular version we are not facing the issue.

**Comment by bcanterbury** [comment]
I had a similar problem for the last month or two.  All the research I found just said node.js wasn't updating itself.  Today I had a revelation.  I looked up the logs for installs in the windows event viewer and found the culprit.

![image](https://github.com/nodejs/help/assets/82123552/7fd0409a-054e-4955-ac90-69f2df614834)

Do something similiar yourself if you have this issue.  look at the event logs and find the path of the installation.
